### DISCUSSION TITLE
🌝一些奇奇怪怪的工具

### DISCUSSION THREAD

**Discussion by TinsFox** [TinsFox/devui-cli#4] [Ideas]
# 奇奇怪怪的工具锦集
> 记录一些神奇的工具

- [All Contributors](https://allcontributors.org/)
- [Renovate](https://github.com/renovatebot/renovate)
- [Issues Helper](https://github.com/actions-cool/issues-helper)
- [PR Welcome](https://github.com/actions-cool/pr-welcome)
- [@ls-lint/ls-lint](https://www.npmjs.com/package/@ls-lint/ls-lint)

**Comment by kagol** [comment]
可以简单介绍下这个工具的作用、使用场景、如何快速上手之类的

**Comment by TinsFox** [comment]
# [All Contributors](https://allcontributors.org/)
作用：通过cli或者是robot添加贡献者到贡献者名单中去如下图，对于对项目做出贡献，包括不限于提出issues，解决issues，发起pr等的同学写入贡献者名单，可以说是给贡献者一份参与感吧，至于荣耀那就看项目的成就了🌝
![image](https://user-images.githubusercontent.com/33956589/142991816-f8800b78-f29d-43cf-a7b4-454a500dc5d4.png)

食用建议：建议是通过cli进行初始化，通过@robot 的方式让rebot发起更新贡献者名单的PR
操作步骤：跟着[文档](https://allcontributors.org/docs/en/overview)不会有什么问题

**Comment by TinsFox** [comment]
# [Issues Helper](https://github.com/actions-cool/issues-helper)
使用场景：在issues在状态发生变化（创建、编辑、添加label等）对issues做出对应的自动化操作
作用：
- 可以对issues的标题、内容等做粗略的检测
- 通过label做一些事情，比如通知issues创建者提供更多信息，通知issues创建者有人在处理issues了
食用建议：对CI/CD有简单的了解，了解Github Action
操作步骤：编写action文件
示例仓库: https://github.com/TinsFox/issue-helper
示例仓库实现效果：https://github.com/TinsFox/issue-helper/issues/18
